<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>IPasswdDialog</class>
  <widget class="QDialog" name="IPasswdDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>240</width>
        <height>154</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>TuxCards Encryption</string>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <widget class="QLabel" name="textLabel3">
          <property name="font">
            <font>
              <italic>true</italic>
            </font>
          </property>
          <property name="text">
            <string>Please enter a password to encrypt</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="mpIEDescription">
          <property name="text">
            <string>'entry name'</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QGridLayout">
          <item row="1" column="1">
            <widget class="QLineEdit" name="mpPasswdLineTwo">
              <property name="echoMode">
                <enum>QLineEdit::Password</enum>
              </property>
            </widget>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="textLabel2">
              <property name="text">
                <string>Retype</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QLineEdit" name="mpPasswdLineOne">
              <property name="echoMode">
                <enum>QLineEdit::Password</enum>
              </property>
            </widget>
          </item>
          <item row="0" column="0">
            <widget class="QLabel" name="textLabel1">
              <property name="text">
                <string>Password</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <spacer name="spacer1">
              <property name="sizeHint">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="mpOkButton">
              <property name="text">
                <string>&amp;Ok</string>
              </property>
              <property name="shortcut">
                <string>Alt+O</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="pushButton2">
              <property name="text">
                <string>&amp;Cancel</string>
              </property>
              <property name="shortcut">
                <string>Alt+C</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>mpPasswdLineOne</tabstop>
    <tabstop>mpPasswdLineTwo</tabstop>
    <tabstop>mpOkButton</tabstop>
    <tabstop>pushButton2</tabstop>
  </tabstops>
  <connections>
    <connection>
      <sender>pushButton2</sender>
      <signal>released()</signal>
      <receiver>IPasswdDialog</receiver>
      <slot>reject()</slot>
    </connection>
    <connection>
      <sender>mpOkButton</sender>
      <signal>clicked()</signal>
      <receiver>IPasswdDialog</receiver>
      <slot>done()</slot>
    </connection>
  </connections>
</ui>
